doc/content/verification_and_validation/val-2f.md

```diff
@@ -0,0 +1,209 @@
+| Parameter | Description                            | Value                                                       | Units                 | Reference                 |
+| --------- | ------------------------------------   | ----------------------------------------------------------- | --------------------- | --------------------- |
+| $\alpha_{t0}^i$ | Pre-factor of trapping rate coefficient | $\frac{D_0}{6 \cdot (1.1\times 10^{-10})^2}$                                           | atoms/s               | [!cite](dark2024modelling) |
+| $\alpha_{r0}^i$ | Pre-factor of release rate coefficient | $10^{13}$                                           | atoms/s               | [!cite](dark2024modelling) |
+| $\epsilon_t^i$ | Trapping energy for two traps     | 0.28                                                        | eV                    | [!cite](dark2024modelling) |
+| $\epsilon_r^1$ | Release energy for trap 1     | 1.15                                                        | eV                    | [!cite](dark2024modelling) |
+| $\epsilon_r^2$ | Release energy for trap 2     | 1.35                                                        | eV                    | [!cite](dark2024modelling) |
+| $\epsilon_r^3$ | Release energy for trap 3     | 1.65                                                        | eV                    | [!cite](dark2024modelling) |
+| $\epsilon_r^4$ | Release energy for trap 4     | 1.85                                                        | eV                    | [!cite](dark2024modelling) |
+| $\epsilon_r^5$ | Release energy for trap 5     | 2.05                                                        | eV                    | [!cite](dark2024modelling) |
+| $\epsilon_r^6$ | Release energy for intrinsic trap     | 1.04                                                        | eV                    | [!cite](dark2024modelling) |
+| $N_1$ | Density for trap 1                          | 4.8$\times 10^{25}$                                         | atoms/m$^3$           | [!cite](dark2024modelling) |
+| $N_2$ | Density for trap 2                             | 3.8$\times 10^{25}$                                         | atoms/m$^3$           | [!cite](dark2024modelling) |
+| $N_3$ | Density for trap 3                             | 2.6$\times 10^{25}$                                         | atoms/m$^3$           | [!cite](dark2024modelling) |
+| $N_4$ | Density for trap 4                             | 3.6$\times 10^{25}$                                         | atoms/m$^3$           | [!cite](dark2024modelling) |
+| $N_5$ | Density for trap 5                             | 1.1$\times 10^{25}$                                         | atoms/m$^3$           | [!cite](dark2024modelling) |
+| $N_6$ | Density for intrinsic trap                             | 2.4$\times 10^{22}$                                         | atoms/m$^3$           | [!cite](dark2024modelling) |
+
```

Comment:
You need to add a paragraph here about the different values of recombination paramaters that you use, why you use different ones, and how you picked these values. You should also describe what is done in the original paper from Dark et al.